src/utils/activation_functions.jl

```diff
@@ -134,37 +134,47 @@ end
 # Sigmoid (invertible if ouput nonzero)
 
 """
-    y = Sigmoid(x)
+    y = Sigmoid(x; low=nothing, high=nothing)
 
- Sigmoid activation function.
+ Sigmoid activation function. Can be shifted and scaled such that output is (low,high].
 
  See also: [`SigmoidInv`](@ref), [`SigmoidGrad`](@ref)
 """
-function Sigmoid(x)
-    y = 1f0 ./ (1f0 .+ exp.(-x))
+function Sigmoid(x; low=nothing, high=nothing)
```

Comment:
`low=0, high=1` no need for if/else, same everywhere

Reply:
As far as I can see, this is correctly implemented.